miniscope_io/stream_daq.py

```diff
@@ -322,12 +333,16 @@ def _fpga_recv(
 
         locallogs.debug("Starting capture")
         try:
-            while not self.terminate.is_set():
+            while 1:
                 try:
                     buf = dev.readData(read_length)
-                except (EndOfRecordingException, StreamReadError, KeyboardInterrupt):
+                except (EndOfRecordingException, KeyboardInterrupt):
                     locallogs.debug("Got end of recording exception, breaking")
                     break
+                except StreamReadError:
+                    locallogs.exception("Read failed, continuing")
+                    # It might be better to choose continue or break with a continuous flag
+                    continue
```

Comment:
Lowered severity of `StreamReadError` so the DAQ keeps running even when the data stream stops temporary.

Reply:
Yes this I think is the core of the pr right? To continue capturing through errors?

Reply:
Yes it was mostly this, adding handling errors around cv2 part, and timeouting puts (not absolutely necessary atm) for the continuous part.